starboard/android/shared/player_components_factory.h

```diff
@@ -195,6 +191,13 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
     return (value + alignment - 1) / alignment * alignment;
   }
 
+  static bool IsForceTunnelModeEnabled() {
+    bool force_tunnel_mode =
```

Comment:
We'd prefer this to be controlled by the web app.  Do you happen to know if there are any particular devices that we should force enable tunnel mode?  We may consider including the devices in our initial experiment.

Please feel free to continue this discussions via email if that's preferred.

Reply:
Thanks for the notification and review. Sorry for my really late response.
I will discuss internally and let you know through email if we have a device that needs to enable tunnel mode for the experiment.